src/Language/PureScript/Bridge/Printer.hs

```diff
@@ -61,13 +60,13 @@ moduleToText settings m = T.unlines $
   "-- File auto generated by purescript-bridge! --"
   : "module " <> psModuleName m <> " where\n"
   : map importLineToText allImports
-  ++ [ ""
-     , "import Prelude"
+  <> [ ""
+     , "import Prelude" -- TODO: need a switch for Foreign.Generic to see if we should import or not and then implementation of the instances
```

Comment:
Is that todo still relevant?

Reply:
this is why we review code 😄 

removed and pushed